docs/source/ko/_toctree.yml

```
@@ -145,8 +145,8 @@
     title: (번역중) bitsandbytes
   - local: in_translation
     title: (번역중) GPTQ
-  - local: in_translation
-    title: (번역중) AWQ
+  - local: quantization/awq.md
```

No need to append the file format at the end, it's causing the doc-builder to fail

```suggestion
  - local: quantization/awq
```